server/repository/src/db_diesel/insurance_provider_row.rs

```diff
@@ -0,0 +1,96 @@
+use super::{
+    insurance_provider_row::insurance_provider::dsl as insurance_provider_dsl, ChangeLogInsertRow,
```

Comment:
Dang, just realised that this is the old style approach with `::dsl` will update...

Reply:
Thanks for that!